tests/entrypoints/test_openai_server.py

```diff
@@ -62,7 +62,7 @@ def zephyr_lora_files():
 
 @pytest.fixture(scope="session")
 def server(zephyr_lora_files):
-    ray.init()
+    ray.init(ignore_reinit_error=True)
```

Comment:
is this something we should push upstream?

Reply:
likely, here first though